docs/components/board/_index.md

```diff
@@ -669,11 +669,12 @@ status = await my_board.status()
 - `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
 - `mode` [(PowerMode)](https://pkg.go.dev/go.viam.com/api/component/board/v1#PowerMode): Options to specify power usage of the board: `boardpb.PowerMode_POWER_MODE_UNSPECIFIED`, `boardpb.PowerMode_POWER_MODE_NORMAL`, and `boardpb.PowerMode_POWER_MODE_OFFLINE_DEEP`.
 - `extra` [(map[string]interface{})](https://go.dev/blog/maps): Extra options to pass to the underlying RPC call.
-- `duration` [(*time.Duration)](https://pkg.go.dev/time#Duration): If provided, the board exits the given power mode after the specified time in nanoseconds has passed.
```

Comment:
Does the asterisk not indicate anything important?

Reply:
Good question here-- I think the asterisk indicates for the function specification that we are passing a pointer variable of the time.Duration value type. I thought that we didn't necessarily have to specify this here, but after going over [this](https://dev.to/mr_destructive/golang-pointers-1bp3#:~:text=So%2C%20this%20is%20how%20*%20and,It's%20that%20simple) I agree that this should be reverted!

Reply:
Done/reverted!